graph/options/options.go

```diff
@@ -60,7 +63,9 @@ func NewOptions(r *http.Request) Options {
 		checkError(err)
 
 		for _, namespace := range namespaces {
-			namespaceNames = append(namespaceNames, namespace.Name)
+			if namespace.Name != NamespaceIstioSystem {
```

Comment:
Maybe a comment here to say that `includeIstio` flag is not used for graph generation but only for metrics?
Else, I'm pretty sure some day someone will look at this code and ask why this "if" doesn't check `includeIstio` as well :)

Reply:
I added a comment here in a different PR...